src/App.js

```diff
@@ -17,15 +17,29 @@ Amplify.configure({
   }
 });
 
-const MyApp = (props) => {
-  const token = props?.authData?.signInUserSession?.idToken?.jwtToken;
-  setToken(token)
+// TO-DO: figure out how to integrate DHIS2 i18n with amplify
+I18n.putVocabulariesForLanguage('en', {
+  'Sign In': 'Login', // Tab header
+  'Sign in': 'Login', // Button label
+  Password: 'Enter your password', // Password label
+  'Forgot your password?': 'Reset Password',
+});
+
+const MyApp = () => {
```

Comment:
How does a user logout? And should we force a logout once the token expires?

Reply:
A user can log out of DHIS2 so we can handle in a different PR to log out if the user logs out of DHIS2.